packages/empty/src/img-empty.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <svg
+    width="79px"
+    height="86px"
```

Comment:
I think these 2 attributes can be removed. Could you remove it and see if it works?